src/main/java/org/cactoos/text/TextOf.java

```diff
@@ -427,11 +403,7 @@ public TextOf(
      */
     public TextOf(final LocalDate date, final DateTimeFormatter formatter) {
         this(
-            () -> formatter.format(
-                ZonedDateTime.of(
-                    date, LocalTime.MIN, ZoneId.systemDefault()
-                )
-            )
+            new TextOfDate(date, formatter)
```

Comment:
@baudoliver7 it seems the `TextOfDate` class is specific enough for justifying not having those constructors in `TextOf` so that users directly uses `TextOfDate`